telemetry/specs/INT.mdk

```diff
@@ -937,6 +1118,7 @@ port ID)
   - C = 0
   - E = 0 (Max Hop Count not exceeded)
   - M = 0 (MTU not exceeded at any switch)
+  - S = 0
```

Comment:
This change and all the changes below should be reverted as well since the S bit is removed.

Reply:
Done